<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flugbuchung</class>
 <widget class="QDialog" name="flugbuchung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>ID: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IDlineEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Airline:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="airlineLineEdit"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>From:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fromDestLineEdit"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>To:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="toDestLineEdit"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Departure:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="fromDateEdit"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Arrival:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="toDateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="priceLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
